### PR TITLE
added renoise

### PR DESCRIPTION
**Pull request by cgadski**
added renoise, a tracker-style DAW. Installs demo version by default.

### CODE REVIEW

**Comment by lethalman** [comment]
Why this new style for writing derivations... couldn't you use a single derivation instead of those two functions?

**Comment by MagneticDuck** [comment]
I like separating the logical parts of a build so that it's easier to understand (for me at least) and you can sometimes change parts without having to rebuild everything.

**Comment by lethalman** [comment]
I find it very hard to follow, and don't understand why you have to make two derivations by moving parts of the first into the second with a different layout.

**Comment by krgn** [comment]
I have tested the package (non-demo version of Renoise) and it works for me, although I had to make one amendment prior to getting it to build: the file name of the download is `rns_3_0_1_reg_x86_64.tar.gz` and the extracted folder `rns_3_0_1_x86_64/`. 

Personally, I'd really like to see this in-tree. Thanks @MagneticDuck!

**Comment by krgn** [comment]
Ah. Found another change needed to make `renoise` aware of jack; libjack2 was missing from its dependencies.

**Comment by krgn** [comment]
Hey, I have simplified the derivation a bit, and put it in a gist: https://gist.github.com/krgn/91d911998ea344409cae

Use it if you like, let me know and I can create a PR with it. Anyhow, would be cool merge this in! :)

**Comment by lethalman** [comment]
@krgn I don't really see the need for the let/in expression other than complicating the code... sorry

**Comment by krgn** [comment]
@lethalman hm, ok. I'll look at some more examples to see what the general style looks like

**Comment by krgn** [comment]
@lethalman I have updated that gist. Would you care to take a look?

**Comment by krgn** [comment]
Note: think it would be good to support the i686 version as well, and clean up that code more.

**Comment by lethalman** [comment]
Better. But phases and unpackPhase are not needed. Also it's better if you update this PR.

**Comment by krgn** [comment]
Not sure I have permission to update. I do have the changes in my fork though. @MagneticDuck might want to comment on what to do with it?

**Comment by lethalman** [comment]
Of course you have, just force push to the branch used for the PR.

**Comment by krgn** [comment]
It is neither my PR nor fork, so I don't think I do.

**Comment by lethalman** [comment]
Ah sorry, open a new PR then :)

**Comment by krgn** [comment]
OK. PR with amendments is #9888

**Comment by Mathnerd314** [comment]
This can be closed because #9888 was merged.
